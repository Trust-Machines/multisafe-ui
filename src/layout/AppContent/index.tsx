import React from 'react';
import {Box} from '@mui/material';
import {grey} from '@mui/material/colors';
import {useTheme} from '@mui/material';

const AppContent = (props: { children: React.ReactNode }) => {
    const theme = useTheme();

    const boxStyles = {
        bgcolor: theme.palette.mode === 'light' ? grey[50] : grey[900],
        color: theme.palette.mode === 'light' ? grey[900] : grey[300],
        height: 'calc(100% - 40px)',
        flexGrow: 1,
        padding: '20px',
        overflow: 'auto'
    }

    return <Box sx={boxStyles}>{props.children}</Box>
}

export default AppContent;