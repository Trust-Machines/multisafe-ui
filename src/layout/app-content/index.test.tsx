import {render} from '@testing-library/react';

import AppContent from './index';

test('1 Default render', () => {
    const view = render(<AppContent><span>children</span></AppContent>);
    expect(view.container).toMatchSnapshot();
});