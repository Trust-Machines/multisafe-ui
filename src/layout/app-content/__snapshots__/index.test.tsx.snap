// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 Default render 1`] = `
<div>
  <div
    class="MuiBox-root css-l1p3en"
  >
    <span>
      children
    </span>
  </div>
</div>
`;
